### PR TITLE
Change the id/primary-key of flow events. Scheduler events are not not triggered multiple times. (~1051)

### PR DESCRIPTION
**Pull request by simplicbe**

### CODE REVIEW

**Comment by simplicbe** [comment]
```sql
DROP TABLE Flow_Event_Queue;
COMMIT;
CREATE GLOBAL TEMPORARY TABLE "admin"."Flow_Event_Queue" (
	"Id" VARCHAR(255) NOT NULL UNIQUE,
	"EventName" VARCHAR(255) NOT NULL,
	"Args" LONG BINARY NOT NULL,
	"CreateDateTime" TIMESTAMP NOT NULL,
	"CreateUserId" INTEGER NOT NULL,
	"Handled" SMALLINT NOT NULL DEFAULT 0,
	"MachineName" VARCHAR(255) NULL,
	"ServiceName" VARCHAR(255) NULL,
	PRIMARY KEY ( "Id" ASC )
) NOT TRANSACTIONAL SHARE BY ALL;
```

**Comment by simplicbe** [comment]
image

File: release-notes.xml [hunk 1]
```diff
@@ -22,4 +22,7 @@
     <Change type="enhancement">Adds a node-generator for public static methods and properties.</Change>
     <Change type="enhancement">Adds a set of generated nodes for System.Convert, System.DateTime, etc.</Change>
   </ChangeSet>
+  <ChangeSet>
+    <Change type="enhancement">The id of a flow-event is not the finger-print of the flow-event name and object-id. This prevents some events from being triggered multiple times.</Change>
```
Comment (simplicbe) [inline release-notes.xml:26]:
```suggestion
    <Change type="enhancement">The id of a flow-event is now the finger-print of the flow-event name and object-id. This prevents some events from being triggered multiple times.</Change>
```

**Comment by simplicbe** [comment]
@MEichhoff please export the sql above with the next simplic framework release (9.9.9.9)

**Review by MEichhoff — APPROVED** [review]

**Pull request merged**